.github/workflows/api_clients_ci.yml

```diff
@@ -80,13 +84,13 @@ jobs:
         with:
           node-version: 20
           cache: "npm"
-          cache-dependency-path: "api/api_clients/typescript/package.json"
+          cache-dependency-path: "packages/typescript/algod_api/package.json"
 
       - name: Build TypeScript ffi package
         run: bun scripts/build ${{ env.CRATE }} typescript
 
-      - name: Test TypeScript client
-        working-directory: api/api_clients/typescript
+      - name: Test algod TypeScript client
+        working-directory: packages/typescript/algod_api
         run: |
           npm install
           npm run test
```

Comment:
I saw that scripts in package json of the ts client were updated to use bun. Should we use bun everywhere at this point then including CI? Also should the README in autogenerated ts client be updated as well to use bun instead of npm?